src/renderer/containers/App.tsx

```diff
@@ -16,7 +17,7 @@ import {AppDispatch, ProtocolType} from '@renderer/types';
 import {displayErrorToast, displayToast} from '@renderer/utils/toast';
 
 const DEFAULT_BANK = {
-  ip_address: '13.57.215.62',
+  ip_address: config.DEFAULT_BANK,
```

Comment:
this needs to be updated to `DEFAULT_BANK_IP_ADDRESS` as well

Reply:
soz - also getting comments from @angle943 - updating config + DEFAULT_BANK_IP_ADDRESS